### DISCUSSION TITLE
请问为什么play安全机制会提示本软件有安全风险

### DISCUSSION THREAD

**Discussion by KanaKiraka** [zhihulite/Hydrogen#136] [General]
![Screenshot_20241201-103346_Google Play 商店.png](https://github.com/user-attachments/assets/f053277c-287a-4a60-9ba5-8a8bda2ce6aa)

**Comment by huajiqaq** [comment]
实际上 **只要是未上架play商店的app都会提示可能有风险** 比如抖音或快手极速版也会提示
